go/consensus/tendermint/full/full.go

```diff
@@ -922,10 +922,12 @@ func (t *fullService) initialize() error {
 		return err
 	}
 	t.registry = scRegistry
-	t.registryMetrics = registry.NewMetricsUpdater(t.ctx, t.registry)
+	if cmmetrics.Enabled() {
+		t.registryMetrics = registry.NewMetricsUpdater(t.ctx, t.registry)
```

Comment:
While we're here, we can probably also just get rid of the `t.registryMetrics` member variable and use a local one instead (as it doesn't seem to be used anywhere else besides getting registered with the service manager on the next line).

Reply:
fixed

Reply:
Thanks!